android/app/src/fdroid/res/layout/fragment_map.xml

```diff
@@ -9,4 +9,11 @@
         tilesource="Cyclemap"
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
+
+    <ZoomControls
+        android:id="@+id/zoomControls"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center|bottom"
+        android:paddingBottom="50dp"/>
```

Comment:
When I said 4dp margin, I meant bottom margin only.
Sorry about the miscommunication.